doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,96 @@
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+**Chapter Purpose**<br>
+This chapter includes tracability from requirements to test cases and a test case execution framework to ensure Kubernetes infrastructure meets the design, feature, and capability expectations of the Reference Model (RM) and RA2. Ultimately, RC2 will reduce the amount of time and cost it takes each operator to on-board and maintain cloud infrastructure and CNFs.
+
+<a name="1.2"></a>
+## 1.2 CNTT RC2 End User Requirements
+
+Telcom service providers / operators are the primary intended audience for CNTT RC-2 results. Specifically those selecting infrastructure and network function technologies to use in their network. The RC2 result artefact(s) should be clear and provide confidence to the service provider that the test results meet the requirements *they have.*
+
+Vendors/Developers are a secondary audience. They should be able to clearly see the mapping from a specific test result (pass or fail) to the requirement.
+
+### Operator/SP Test Result Requirements
+* Clickable links from test cases to requirements
+* Pass, Fail, or Skipped for every test
+* Reason for failures
+* Reason for skipping
+* Format supporting clickable links (eg. HTML)
+* Provide a stable set of point-in-time requirements and tests to achieve conformance
+* Testing tools allow users to select between validation of mandatory and optional requirements
+* Enable clear tracability with versioning to know which requirements have and haven’t been covered and track changes over time
+* Tests must be available to run locally
+* Testing tools must produce machine-readable result formats
+
+### Vendor Test Result Requirements
+* Clear mapping between requirements and test results
+* Enable clear tracability to know which requirements have and haven’t been covered and track changes over time
+* Failures should provide additional content to inform the user where or how the requirement was violated without having to read the test
+* Expected preconditions and environment requirements for any test tooling must be defined
+
+<a name="1.3"></a>
+## 1.3 Scope
+
+This document covers aspects of CNTT conformance for both Kubernetes based cloud infrastructure and workloads. The document will cover the following topics:
+
+- Identify in detail the requirements of test-cases (mapped from RA2 and RM)
+- Test criteria that shows a certain capability or feature of the system-under-test exists and behaves as expected
+- An E2E framework for conformance of Kubernetes infrastructures and workloads, including specification for conformance test infrastructure (lab environment and tools)
+- Analysis to identify where the gaps are in the industry for implementing conformance test objectives (tooling, methods, process, etc)
+
+**Not in Scope**
+- Functional testing / validation of the application provided by the workload is outside the scope of this work
+- Testing to confirm specific versions of APIs is out of scope
```

Comment:
Query on this one - I get we don't want to test for ***a*** specific API version, but given the RA2 spec about being within the k8s API policy of n-2 we should be testing the API to check the version used is within that range?

Reply:
good point, not sure exactly how to phrase this though. Do you have any ideas or should I bring it up in the next meeting?

Reply:
Worth bringing it up (I can't attend I'm afraid - off on holiday).  My suggestion would be to remove it, or replace with a generic comment saying testing for anything not included in RM/RA2 isn't included.  The example I gave of testing for n-2 versions is covered by the in-scope comment `Identify in detail the requirements of test-cases (mapped from RA2 and RM)` so it's whether we want something to cover in more detail what we don't test.